fondant/data_io.py

```diff
@@ -119,6 +116,7 @@ def _create_write_dataframe_task(
         write_task = dd.to_parquet(
             df, remote_path, schema=schema, overwrite=False, compute=False
         )
+        logging.info(f"Creating write task for: {remote_path}")
```

Comment:
Nice, I think we add to add more logging to make debugging easier. Maybe it's better to have this logging at the `write_index` and `write_subset` level? This way you could log both the name of the subset and the path